Framework/Algorithms/src/RemovePromptPulse.cpp

```diff
@@ -63,15 +63,7 @@ double getMedian(const API::Run &run, const std::string &name) {
 }
 
 void getTofRange(MatrixWorkspace_const_sptr wksp, double &tmin, double &tmax) {
-  DataObjects::EventWorkspace_const_sptr eventWksp =
```

Comment:
The old implementation has a mistake in it, but the new one ignores the fact that `EventWorkspace` can have events outside of the binned range. That is what this was supposed to handle. I think changing line 69 to
```
const bool isEvent = bool(eventWksp);
```
would fix it.